site/operational-guide.markdown

```diff
@@ -0,0 +1,384 @@
+  scaling events
+
+### Scale-out
+
+On additional peer:
+
+    weave launch --observer <fixed cluster peers>
+
+> Author's Note: `--observer` prevents a clique forming amongst a
+> group of dynamically added peers if they become partitioned from the
+> fixed cluster after having learned about each other. We only specify
+> the fixed cluster peers as targets to avoid the machinery which is
+> responding to the scale-out event from having to keep track of
+> dynamic peers
+
+### Scale-in
+
+On peer to be removed:
+
+    weave reset
+
+## Uniform Dynamic Cluster
+
+> Author's Note: TBD - this will talk about `--ipam-seed` and how it
+> can be used with e.g. puppet to a) stand up an initial cluster and
+> b) add arbitrary new nodes without requiring external
+> synchronisation
+
+# Administrative Tasks
+## Configuring Weave to Start Automatically on Boot
+
+`weave launch` runs all weave's containers with a Docker restart
+policy of `always`, so as long as you have launched weave manually
+once and your system is configured to start Docker on boot then weave
+will be started automatically on system restarts.
+
+If you're aiming for a non-interactive installation, you can use
+systemd to launch weave after Docker - see [systemd docs](TODO) for
+details.
+
+> Author's Note: I'm going to pull the complete systemd doc into this
+> section
+
+## Recovering Lost IPAM Space
+
+The recommended way of removing a peer is to run `weave reset` on that
+peer before the underlying host is decommissioned or repurposed - this
+ensures that the portion of the IPAM allocation range assigned to the
+peer is released for reuse. Under certain circumstances this operation
+may not be successful, or indeed possible:
+
+* If the peer in question is partitioned from the rest of the network
+  when `weave reset` is executed on it
+* If the underlying host is no longer available to execute `weave
+  reset` - for example due to a hardware failure or other unplanned
+  termination
+
+In either case the remaining peers will all consider the dead peer's
+address space to be unavailable even after it has remained unreachable
+for prolonged periods; there is no universally applicable time limit
+after which a peer could decide unilaterally that it is safe to
+appropriate the space for itself, and so an adminstrative action is
+required to reclaim it.
+
+The `weave rmpeer` command is provided to perform this task, and must
+be executed on one of the remaining peers. That peer will take
+ownership of the freed adress space.
+
+## Rolling Upgrades
+
+Protocol versioning and feature negotation are employed in Weave Net
+to enable incremental rolling upgrades - each release maintains the
+ability to speak to the previous version (at a minimum), and connected
+peers only utilise features which both support. The general upgrade
+procedure is as follows:
+
+On each peer in turn:
+
+* Download the new weave script to a temporary location e.g.
+  `/path/to/new/weave`
```

Comment:
Temporary location doesn't seem to be necessary.  Just get the new file, `weave setup`, `weave stop` and `weave launch`